scripts/s3bucketsize/s3bucketsize.py

```diff
@@ -0,0 +1,101 @@
+"""
+This Python script calculates the total size of items in specified S3
+buckets and logs the results in a CSV file.
+
+The function runs once per day and performs the following steps:
+
+1. Retrieves the list of S3 bucket names from the environment variable 
+   `BUCKET_NAMES` and `OUTPUT_BUCKET` or from the .env file.
+2. Calculates the total size of objects in each specified S3 bucket.
+3. Appends the calculated sizes along with the current date to a CSV file.
+4. Checks if the CSV file already exists in the specified output S3 bucket
+   (defined by the `OUTPUT_BUCKET` environment variable).
+5. If the CSV file exists, downloads it, appends the new data, and uploads the
+   updated file back to the S3 bucket.
+6. If the CSV file does not exist, creates a new CSV file with the new data and
+   uploads it to the S3 bucket.
+
+Environment Variables:
+- BUCKET_NAMES: A comma-separated list of S3 bucket names to calculate the
+  total size for.
+- OUTPUT_BUCKET: The name of the S3 bucket where the CSV file will be saved.
+
+CSV File Format:
+- The CSV file contains the following columns: Date, BucketName, SizeInBytes.
+
+Example CSV Content:
+Date, BucketName, SizeInBytes
+2023-10-01T00:00:00,example-bucket-1,123456789
+2023-10-01T00:00:00,example-bucket-2,987654321
```

Comment:
Super minor comment to add the new ItemCount field you put in for me
```suggestion
CSV File Format:
- The CSV file contains the following columns: Date, BucketName, SizeInBytes, ItemCount.

Example CSV Content:
Date, BucketName, SizeInBytes, ItemCount
2023-10-01T00:00:00,example-bucket-1,123456789,1234
2023-10-01T00:00:00,example-bucket-2,987654321,5678
```

Reply:
Ah yes, sounds good